CMakeLists.txt

```diff
@@ -66,7 +68,8 @@ set(CLANG_WARNINGS
     -Wpedantic # warn if non-standard C/C++ is used
     -Wlanguage-extension-token
 )
-target_compile_options(mega65libc PRIVATE -mcpu=mos65c02 -Os ${CLANG_WARNINGS})
+add_compile_options(-mcpu=mos65ce02 -Os ${CLANG_WARNINGS})
```

Comment:
why setting these compile options globally? target_compile_options would still be the preferred modern cake way, I assume?

Reply:
To avoid setting this for all tests. Strangly just setting `add_compile_options()` gives me a different set of warnings than if setting it for `mega65libc`. Not sure it matters too much in this setup.